pages/agent/v3/elastic_ci_stack_for_ec2_mac/autoscaling_mac_metal.md.erb

```diff
@@ -215,3 +215,7 @@ configuration has a *License type* of `Cores`.
 Ensure your AMI has been [configured to auto-login as the `ec2-user`](#step-2-build-an-ami)
 in the GUI. To allow your pipelines to use Xcode and the iOS Simulator the
 Buildkite Agent launchd job configuration requires an `Aqua` session type.
+
+### What user does the agent run as?
+
+The Buildkite agent runs as ec2-user.
```

Comment:
Is this `ec2-user` or is this "the EC2 user"?

Reply:
According to Keith's notes, "Elastic Mac for AWS, the ec2-user"

Reply:
A quick google suggests there actually is something called the ec2-user. E.g https://stackoverflow.com/a/60029049/2291838

Reply:
```suggestion
The Buildkite agent runs as `ec2-user`.
```